config.yaml

```diff
@@ -166,10 +166,10 @@ options:
       the Juju version is dynamically retrieved.
     type: string
     default:
-  jem-url:
+  bundleservice-url:
     type: string
     default: ''
```

Comment:
We should probably have a valid default here: `https://api.jujucharms.com/bundleservice/`